pinot-core/src/main/java/org/apache/pinot/core/transport/server/routing/stats/ServerRoutingStatsManager.java

```diff
@@ -198,6 +198,10 @@ private void updateStatsUponResponseArrival(String serverInstanceId, long latenc
     }
   }
 
+  public Map<String, ServerRoutingStatsEntry> getServerRoutingStats() {
```

Comment:
can you add tests for this in `ServerRoutingStatsManagerTest` to validate that it looks as expected?

Reply:
Done